client/src/components/Navbar.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+import * as FaIcons from 'react-icons/fa'
+import * as AiIcons from 'react-icons/ai'
+import { Link } from 'react-router-dom'
+import { SidebarData } from './SidebarData'
+import './Navbar.css'
+import { IconContext } from 'react-icons'
+
+// componentDidUpdate (prevProps) {
+//     if (prevProps.location.key !== this.props.location.key) {
+//         this.setState({
+//             isFormSubmitted: false,
+//         })
+//     }
+// }
+
+function Navbar() {
+  const [sidebar, setSidebar] = useState(false)
+
+  const showSidebar = () => setSidebar(!sidebar)
+
+  return (
+    <>
+      <IconContext.Provider value={{ color: '#fff' }}>
+        <div className='navbar'>
```

Comment:
I see that you're using css classes instead of styled-components for styling. Can you refactor everything to use styled-components, so that the styling is consistent across all pages?

Reply:
For example, instead of having .navbar in a css file, try declaring a styled component like this:
```
const NavBarDiv = styled.div`
    background-color: #152238;
    height: 80px;
    display: flex;
    justify-content: flex-start;
    align-items: left;
`
```

Then, you can replace `<div className='navbar'></div>` with `<NavBarDiv></NavBarDiv>`